usr/src/uts/common/fs/zfs/sys/refcount.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
  * particular object, use FTAG (which is a string) for the holder_tag.
  * Otherwise, use the object that holds the reference.
  */
-#define	FTAG ((char *)__func__)
+#define	FTAG ((char *)(uintptr_t)__func__)
```

Comment:
Otherwise:
```
      cast from 'const char *' to 'char *' drops const qualifier
      [-Werror,-Wcast-qual]
        VERIFY3U(0, ==, dmu_bonus_hold(os, obj, FTAG, &db));
                                                ^
```
I suppose a better solution might be for `FTAG` to cast to a `const char*`, but that doesn't work because too many functions, like `dmu_bonus_hold`, expect a non-constant argument.  Fixing that would be a more intrusive change.